types/index.d.ts

```diff
@@ -1884,6 +1884,7 @@ export declare namespace Knex {
     boolean(columnName: string): ColumnBuilder;
     date(columnName: string): ColumnBuilder;
     dateTime(columnName: string, options?: Readonly<{useTz?: boolean, precision?: number}>): ColumnBuilder;
+    datetime(columnName: string, options?: Readonly<{useTz?: boolean, precision?: number}>): ColumnBuilder;
```

Comment:
what is it for? isn't this overload fully equivalent to the previous one?

Reply:
I think that the problem was mentioned in #3944 where the same method is aliased as ```dateTime```and ```datetime``` but only the definition for ```dateTime``` is available in TS defs while only the ```datetime``` is mentioned in documentation so i added ```datetime``` to the TS defs.

Reply:
ah, gotcha, I missed the difference in casing. thank you